### PR TITLE
fix #9

### LINKED ISSUE #9: `status` accepting multiple job ids
for example

```bash
cromwell status job_id_1 job_id_2 job_id_3 ...
```

### PR DESCRIPTION
**Pull request by SHuang-Broad**

### CODE REVIEW

**Comment by SHuang-Broad** [comment]
As discussed, also supports `abort` now.

Docs updated as requested.

File: cromwell [hunk 1]
```diff
@@ -93,12 +95,12 @@ else
   if [ -n "$2" ]; then workflowID=$2; else workflowID=$(cat ${CROMWELL_LAST_WORKFLOW_FILE} ); fi
 fi
 echo "using workflow-id == $workflowID" 1>&2
-if [ "$1" = "status" ] ; then
+if [ "$1" = "status" ] || [ "$1" = "abort" ] ; then
    if [[ "$#" -le 2 ]] ; then
     status $workflowID
```
Comment (lbergelson) [inline cromwell]: won't this do the wrong thing for abort now?

Comment (SHuang-Broad) [reply cromwell]: oops! right. now fixed

**Comment by SHuang-Broad** [comment]
Oh I've been using it for a week now..

**Pull request merged**